frontend/src/store/slice/Experiments/ExperimentsUtils.ts

```diff
@@ -11,7 +11,10 @@ export function convertToExperimentListType(
 ): ExperimentListType {
   const experimentList: ExperimentListType = {}
   Object.entries(dto).forEach(([uid, value]) => {
-    experimentList[uid] = convertToExperimentType(value)
+    experimentList[uid] = {
+      ...convertToExperimentType(value),
+      frameRate: convertToExperimentType(value).frameRate
+    }
```

Comment:
Please refactor here.
Update `ExperimentType` itself.
(`frameRate`'s value should be set in the `convertToExperimentType` method.)

Reply:
@ReiHashimoto I fixed it, please help me check